### ISSUE TITLE
Workflow for integration

### ISSUE THREAD

**Issue by Anticept** [wiremod/advdupe2#1]
To facilitate efficient integration of advdupe2, we're following a simple workflow plan. Basically, we need to:
- [x] import advdupe2 and examine the codebase, and make sure it's worth the effort to integrate into wiremod. 
- [x] re-license it to apache 2; the permission from the developers have already been obtained.
- [x] fix the codebase such that we can remove the compatibility workarounds and hacks placed into the Wiremod codebase.

Once this is done then the integration milestone will be considered complete, and this issue thread closed.

A new issue thread will then be opened for deployment, which will serve the purpose of pushing adv dupe 2 to the workshop.

Please do not use this thread for chatter or issue reporting. It exists to comment on the status of this milestone, and to determine when it is appropriate to move on to deployment.
